va/va.h

```diff
@@ -1401,6 +1408,19 @@ typedef union _VAConfigAttribValContextPriority {
     uint32_t value;
 } VAConfigAttribValContextPriority;
 
+/** brief Attribute value VAConfigAttribEncDeltaQpPerBlock */
+typedef union VAConfigAttribValEncDeltaQpPerBlock {
+    struct {
+        /** \brief whether to support dela qp per block */
+        uint32_t delta_qp_support         : 1;
+        /** \brief supported size of delta qp block */
+        uint32_t log2_delta_qp_block_size : 4;
+        /** \brief reserved bit for future, must be zero */
+        uint32_t reserved                 : 27;
+        } bits;
+        uint32_t value;
+} VAConfigAttribValEncDeltaQpPerBlock;
```

Comment:
same as above

Reply:
Rename it to VAConfigAttribValEncPerMBControl